### PR TITLE
Add solution for Murder Mystery

### PR DESCRIPTION
**Pull request by wiredlime**

### CODE REVIEW

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 1]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
 ---
 
-[Back to README](../README.md)
+
+# Murder Mystery
+Init: Open file README.md
+Open your terminal
+
+ 
+#### 00
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md]: Please use markdown linter. 
number of `#` must follow order #, ##, ###, #### .
Step should have work step then number

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 2]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+ 
+#### 00 
+1. `cd desktop/cmystery`
+2. `cat step0`
+
+3. `Cmd +N`: Open a new terminal to navigate around and testing new commandlines. 
+
+By the time, there are two opened Terminal, one for reading instructions one for testing commandlines and navigate files.
+
+![](https://i.imgur.com/jPQwZr5.png)
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md]: As a web developer, one must use the alternated text regularly

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 3]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+
+
+Check up folder `Detective Handbook`
+Check up folder `mystery`
+
+
+#### 01
+
+------
+From: clmystery/
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md]: Should be to, not from

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 4]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+------
+
+1. `cat step1`
+2. `less mystery/crimescene` : to view content in different viewer.
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md]: less is not to view content on different viewer. I think you are trying to tell user to view content in different viewer, however, your wording here looks like an explanation for the `less m/c`

Comment (lukehoang1905) [reply module-1/m14-wev-dev-environment/muder-mystery.md]: Less is a command line utility that displays the contents of a file or a command output, one page at a time

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 5]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+3. `q` : exit viewer
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md]: viewer? reading screen? view mode? 
I think file view mode is better

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 6]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+
+4. `touch clues.txt`
+
+5. `grep CLUE mystery/crimescene > clues.txt`
+
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md:43]: Missing header : `Outcome check`

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 7]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+**CLUES:**
+*CLUE: Footage from an ATM security camera is blurry but shows that the perpetrator is a tall male, at least 6'.
+CLUE: Found a wallet believed to belong to the killer: no ID, just loose change, and membership cards for AAA, Delta SkyMiles, the local library, and the Museum of Bash History. The cards are totally untraceable and have no name, for some reason.
+CLUE: Questioned the barista at the local coffee shop. He said a woman left right before they heard the shots. The name on her latte was Annabel, she had blond spiky hair and a New Zealand accent.*
+
+6. `git add .``
+7. `git commit -m "Upload clues"``
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md]: trailing ` found at end of line

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 8]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+5. `ls mystery/streets/Hart_*`
+6. `less mystery/streets/Hart_Place`
+7. `q`
+
+#### 04
+
+
+1. `cat step4`
+2. `touch interview.txt`
+3. `head -179 mystery/streets/Buckingham_Place | head -40 mystery/streets/Hart_Place > interview.txt `
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md:78]: I see you have stopped giving explanation per course. However, for pipelined command like this, learner may need a tiny description.

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 9]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+6. `git add .`
+7. `git commit -m "Upload interview files"`
+8. `git push `
+
+#### 05
+
+1. `git checkout step-5`
+2. `cat step5`
+
+3. `at mystery/interviews/interview-47246024 | cat mystery/interviews/interview-699607 > interview.txt > interview_info.txt`
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md:94]: description

File: module-1/m14-wev-dev-environment/muder-mystery.md [hunk 10]
```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
...
+7. `git merge step-5 -m "Merge step5"`
+
+#### 06
+
+**CLUES:**
+*Describes it as a blue Honda, with a license plate that starts with "L337" and ends with "9". Tall Male 6'*
+
+1. `cat step6`
+
+2. `grep -i "L337.*9" -A6 mystery/vehicles | grep -i Honda -B2 -A5 | grep -i Blue -B3 -A4 | grep -i "6'" -B5 -A2 > suspects.txt`
```
Comment (lukehoang1905) [inline module-1/m14-wev-dev-environment/muder-mystery.md:106]: description

**Comment by wiredlime** [comment]
Content revised, please review and merge

**Review by lukehoang1905 — APPROVED** [review]
LGTM

**Pull request merged**